src/main/common/filter.c

```diff
@@ -42,15 +42,15 @@ FAST_CODE float nullFilterApply(filter_t *filter, float input)
 
 // PT1 Low Pass filter
 
-void pt1FilterInit(pt1Filter_t *filter, uint8_t f_cut, float dT)
+void pt1FilterInit(pt1Filter_t *filter, uint16_t f_cut, float dT)
 {
     float RC = 1.0f / ( 2.0f * M_PI_FLOAT * f_cut );
     filter->k = dT / (RC + dT);
 }
 
 FAST_CODE float pt1FilterApply(pt1Filter_t *filter, float input)
 {
-    filter->state = filter->state + filter->k * (input - filter->state);
+    filter->state += filter->k * (input - filter->state);
```

Comment:
Actually old code is slightly easier to read here (it is closer to mathematical representation)...

Reply:
Wicked of me.  No problem to revert to classic way.
Don't mind sequence of filters either.